attacked/server.py

```diff
@@ -0,0 +1,56 @@
+import socket
+import non_update_functions
+import logging
+import functions
+from colorama import Fore
+from common.const import *
+from common import protocol
+
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+def run_server() -> socket.socket:
+    server_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    server_socket.bind(SERVER_ADDRESS)
+    logging.info(f"[RUNNING] SERVER IS UP AND RUNNING AT {SERVER_ADDRESS}." + Fore.MAGENTA)
+    server_socket.listen()
+    return server_socket
+
+
+def connect_client(server_socket: socket.socket):
+    logging.info("[LISTENING] WAITING FOR CLIENTS...")
+    client_socket, client_address = server_socket.accept()
+    logging.critical("[NEW CONNECTION] {client_address} CONNECT TO SERVER")
+    return client_socket, client_address
+
+
+def handle_request(request_cmd: str, request_params: str, client_address: ADDR):
+    logging.info(f"{client_address} sent the command "

```

Comment:
At it would be printed alot consider makign something for compact   
```python 
logging.info(f"[{client_address: 20}] {request_cmd}({request_params})")
```